setup.py

```diff
@@ -75,12 +75,12 @@
     ],
 
     install_requires=[
-        "greenlet==0.4.10",
-        "requests==2.2.1",
+        "greenlet>=0.4.10",
+        "requests>=2.2.1",
```

Comment:
I think it would better to put upper bounds to be safe, but feel free to ignore this suggestion. For example Requests uses semver: http://docs.python-requests.org/en/master/dev/philosophy/#semantic-versioning

There is also `~=` operator if you prefer it: https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
Using the `~=` will still downgrade if the minor version doesn't match so I'm not sure but an upper bound sounds like a very good idea.

What should the upper bound be? I can bump on our end to latest, try it out and put that as the upper bound.

Reply:
For requests I think it's safe to put `< 3` as they are using SemVer - for others I guess it depends what versioning scheme they are using?

Reply:
I'm also all for adding upper bounds. Of course we need to update the tracker more often, but last time unbounded dependencies caused a lot of pain as we simply didn't know what deps were used last time.

Reply:
+1 for upper bounds

Reply:
@razius If we can get upper bounds on all these, this should be ready for merging.

Reply:
@mhadam , I'll try to find some time this weekend.